modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCFeatureSourceOnlineTest.java

```diff
@@ -690,4 +694,19 @@ public void testMixedEncodeIn() throws Exception {
             fail("Unexpected dialect, supports basic or prepared, but was a : " + dialect);
         }
     }
+
+    /**
+     * Online tests for String functions along with Like operator
+     *
+     * @throws Exception
+     */
+    public void testStringFunction() throws Exception {
+        assumeTrue(dataStore.getFilterCapabilities().supports(FilterFunction_strToLowerCase.class));
```

Comment:
Hum... these tests are based on Junit3, while the assume you pulled is from Junit 4 (the jar contains both). Hmm... maybe it should be done in a different way, like this for example:
https://github.com/geotools/geotools/blob/865cc72a90b07841a4c871d1b07fcc8e5defd35b/modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISHStoreOnlineTest.java#L55

Reply:
understood, on it